packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -22,6 +23,45 @@ import PageAttributes from '../page-attributes';
 import MetaBoxes from '../../meta-boxes';
 import PluginDocumentSettingPanel from '../plugin-document-setting-panel';
 
+const PARTIAL_PATHS = [
```

Comment:
That's one of the hardest challenges of this approach -- deciding what stylesheets should be converted.
> The idea of hard-coding a few partial paths doesn't seem like it would be a very effective solution.

I agree I used hard codding of these styles as a simpler way to test the approach. My idea is for the server to pass a setting named "media_query_rewrite_paths" containing the paths of style sheets that should be rewritten. I guess some of these styles will be common to the styles we use in for the editor styles writing rewrite.

Reply:
Currently these only exist as individual stylesheets in the plugin though. In core they're all bundled together with styles that we don't want to change at all such as wp-admin, editor and component ones. So the solution will need to include making sure all our post-content-relevant styles (or all our full-site-editing-relevant styles?) are bundled as a separate stylesheet in core, and then we'll probably also want to include the theme stylesheet.